SpatialGDK/Documentation/content/map-travel.md

```diff
@@ -0,0 +1,120 @@
+# [Map travel](https://docs.unrealengine.com/en-us/Gameplay/Networking/Travelling)
+
+> Note: This doc is intended for advanced users only.
```

Comment:
doc -> topic 

topic reads better. Go for "page" if not. "doc" is super informal and makes it look like notes rather than the excellent info source it is.